tests/e2e-udx/udx-java/setup-vdb/base/setup-vdb.yaml

```
@@ -0,0 +1,38 @@
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: v-udx-java
+spec:
+  image: kustomize-vertica-image
+  communal:
+    includeUIDInPath: true
+  local:
+    requestSize: 100Mi
+  dbName: vertdb
+  encryptSpreadComm: vertica
+  subclusters:
+    - name: sc1
+      size: 1
+  kSafety: "0"
+  certSecrets: []
+  imagePullSecrets: []
+  # volumes:
+  #   - name: downloads
+  #     hostPath:
+  #       path: /host
+  # volumeMounts:
+  #   - mountPath: /host
+  #     name: downloads
```

We need to include an empty volumes and volumeMounts field. There are some cases where we add those in, so an empty field needs to be inplace for kustomize to work.
```suggestion
  volumes: []
  volumeMounts: []
```